src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -93,9 +93,11 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   Finally, library projects and non-executable projects have awkward interactions here so they are excluded.-->
   <PropertyGroup Condition="'$(UseCurrentRuntimeIdentifier)' == ''">
+    <!-- This excludes MTP projects, as they are normal console apps -->
+    <_IsVSTestOnlyTestProject Condition="'$(IsTestProject)' == 'true' AND '$(IsTestingPlatformApplication)' != 'true'">true</_IsVSTestOnlyTestProject>
```

Comment:
TODO: Ensure that this is even late enough to read `IsTestingPlatformApplication`. It's not invalid to set it in Directory.Build.targets. So, if this is imported *before*, it may not work as expected.